### DISCUSSION TITLE
Which US state is known for its peaches?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#324] [Galaxy Trivia] [answered]
**Hint:** 4th state

**Comment by ParanoidUser** [comment]
*Help wanted:*  @jadehamel, @Malinowsk, @awesomelewis2007, @KR-Ravindra, @ZH1995

![image](https://user-images.githubusercontent.com/5120290/225927467-55bfa48e-df6d-4196-a911-0d3dfff4b9ed.png)

**Comment by thuongtruong1009** [comment, marked as answer]
Georgia is known as the "Peach State" and is famous for its peaches. The state produces a variety of peaches, including clingstone and freestone peaches, and boasts a long history of peach cultivation. Georgia's climate, with hot summers and mild winters, is well-suited for growing peaches, and the state's peach orchards produce a significant portion of the peaches consumed in the United States. The state celebrates its peach production with several peach festivals, including the Georgia Peach Festival, which takes place in multiple cities across the state every June.

**Comment by 0x4248** [comment]
South Carolina is known for its peaches

**Reply by ParanoidUser** [reply]
Nice try. Keep it up! 😉

**Comment by teafoot** [comment]
Georgia :)
